blockchain/blockchain.go

```diff
@@ -0,0 +1,182 @@
+//go:generate abigen --abi ../resources/Agent.abi --pkg blockchain --type Agent --out agent.go
+
+package blockchain
+
+import (
+	"bytes"
+	"crypto/ecdsa"
+	"encoding/hex"
+	"encoding/json"
+	"github.com/coreos/bbolt"
```

Comment:
These aren't in standard gofmt/goimports stanzas, where 3rd party is separated from 1st party is separated from project-local imports. Suggest we standardize on the community standard.